server/punishments.ts

```diff
@@ -748,9 +748,14 @@ export const Punishments = new class {
 			Monitor.log(`[CrisisMonitor] Autolocked user ${name} has public roomauth (${roomauth.join(', ')}), and should probably be demoted.`);
 		}
 
-		const ipStr = typeof user !== 'string' ? ` [${(user as User).latestIp}]` : '';
-		const roomid = typeof room !== 'string' ? (room as Room).roomid : room;
-		Rooms.global.modlog(`AUTO${punishment.replace('ED', '')}: [${userid}]${ipStr}: ${reason}`, roomid);
+		const logEntry: ModlogEntry = {
+			action: `AUTO${punishment.replace('ED', '')}`,
```

Comment:
The `.replace` here feels error-prone.

Reply:
The punishment is hardcoded to be one of `NAMELOCKED`, `WEEKLOCKED`, `NAMEWEEKLOCKED`, or `LOCKED` above. I suppose I could manually add the ED in the monitor.

Reply:
I see, not as urgent then, but I think it would be clearer what is going on